languages/java/exercises/concept/conditionals-ternary/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+In this exercise, you'll be implementing rules for calculating the total salary of a empyoyee in a month. The International Siderurgy Company (ISC) needs help to calculate the salary for the employees, given that different factors can alter the final wage value for each employee.
+
+You have three tasks, but keep in mind that if/else statements should not be used.
```

Comment:
Maybe make it slightly more explicit what is expected:

```suggestion
You have three tasks and you should use the ternary operator instead of if/else statements to implement them.
```